src/presentation/components/DepositModal/DepositModal.tsx

```diff
@@ -0,0 +1,88 @@
+import { useEffect, useState } from 'react';
+import { Button, TextField, Typography } from '@material-ui/core';
+import { ReactComponent as Card } from 'app/assets/CardPicture.svg';
+import { BaseModal } from 'infrastructure/components/Modal/Modal';
+import { useAppSelector } from 'app/hooks/reduxHooks';
+import {
+  useGetBalanceMutation,
+  useInitiateDepositMutation,
+} from 'infrastructure/services/deposit/DepositService';
+import { CustomLoader } from 'infrastructure/components/CustomLoader/CustomLoader';
+import { SuccessVerification } from '../CC/Verification/SuccessVerification';
+import styles from './DepositModal.module.scss';
+
+export const DepositModal = ({ isOpen, handleClose, depositSize, fee }: DepositModalProps) => {
+  const [cvv, setCvv] = useState<number>(0);
+  const [error, setError] = useState('');
+
+  const { defaultCreditCard } = useAppSelector(state => state.creditCard);
+  const [initiateDeposit, { isLoading, isSuccess, isError }] = useInitiateDepositMutation();
+  const [getBalance] = useGetBalanceMutation();
+
+  const handleOnClick = async () => {
+    const response = await initiateDeposit({
+      cvv: cvv,
+      cardId: Number(defaultCreditCard.id),
+      size: depositSize,
+    });
+    console.log('response: ', response);
+    setTimeout(() => {
+      getBalance();
+    }, 30000);
```

Comment:
console log here and is this timeout needed?

Reply:
No it isn't . I already removed both. Thanks